Spec/RestClient.swift

```diff
@@ -839,6 +839,7 @@ class RestClient: QuickSpec {
                         }
                         
                         expect(testHTTPExecutor.requests).to(haveCount(Int(1 + options.httpMaxRetryCount)))
+                        expect((testHTTPExecutor.requests.count) < (_fallbackHosts.count + 1)).to(beTrue())
```

Comment:
@EvgenyKarkan on that line you're checking the `_fallbackHosts`, not the `options.fallbackHosts`. You already verifying the number of requests of the `testHTTPExecutor` and the `_fallbackHosts` is a local var of the test suite that we control, so I think that line is useless.